# Project Summary

We will develop new bioinformatics pattern detection algorithms to infer developmental trajectories and cell fate decisions from both bulk and single-cell RNA-sequencing datasets from distinct batches. Gene expression data from healthy human tissues combines signal from cell fate trajectories, cell cycle, and cell-type specific variation. Single cell RNA-sequencing is a promising data modality to measured these distinct processes. Yet, computational techniques are essential to infer the transcriptional patterns that represent the underlying bioloigcal dynamics. We have developed a sparse Bayesian non-negative matrix factorization (NMF), CoGAPS, that that can simultaneously distinguish dynamic trajectories and individual variation in gene expression data. The algorithm models the underlying estimates of transcript abundence, transcriptional variance, and sparsity characteristic of single-cell RNA-sequencing data. Parallelization of CoGAPS across genes enabled genome wide analysis on small study designs, making it the first NMF algorithm to converge robustly for genome-wide data without requiring compaction. Additional modeling of sparsity in distinct single cell and bulk sequencing technologies will also enable unprecidented, multimodal data integration. We will benchmark the efficiency and sparsity models on time-course, bulk and single cell data in matched conditions provided by the Human Cell Atlas. Algorithms developed will be further optimized by incoprorating biological and technical advances through collaboartion with the proposed collaborative network and broader consortium. 
