# Project Summary

We will develop new bioinformatics pattern detection algorithms to assess variation and reproducibility between bulk and single-cell RNA-sequencing datasets from distinct batches. Gene expression data from healthy human tissues combines signal from individual and tissue specific variation. Parsing complex biological processes is further complicated by technical artifacts in data from diverse sources and platforms. Standard batch correction techniques often overcorrect genomics data, resulting in poor estimation of biological variation. We have developed a sparse Bayesian non-negative matrix factorization (NMF), CoGAPS, that that can simultaneously distinguish batch and biological variation in gene expression data. Parallelization of CoGAPS across genes enabled genome wide analysis on small study designs, making it the first NMF algorithm to converge robustly for genome-wide data without requiring compaction. Here, we will further implement parallelization and platform-specific hyperparameters to optimize CoGAPS for single cell RNA-sequencing. Applying the resulting algorithms to normal samples in TCGA and GTeX will assess biological reproducibility in human samples across multiple tissues and individuals. Complementarily, statistical analysis for systematic comparison of distinct sequencing technologies across biologically matched samples will dissect the relationship between bulk RNAseq and different single cell sequencing techniques. In particular, we will benchmark the effects of sparsity in the different data types with regard to both technical and biological implications. These aims work to establish best practices for distinguishing technical, individual, and tissue specific variation for the Human Cell Atlas.

* Revise to reflect focus on time-course analysis
* Revise to emphasize multi-modal data integration w/ CoGAPS
